### ISSUE TITLE
Environmental variables in logging Firelens sidecar

### ISSUE THREAD

**Issue by sekamaneka** [aws/copilot-cli#2997]
Hello,

is this supported? I could find only references to the normal/storage sidecar. My Firelens sidecar doesn't have any ENV variables.. The COPILOT* ones are missing and the TEST one defined below is also missing.

```
logging:
  image: <>.dkr.ecr.eu-central-1.amazonaws.com/fluentbit:latest
  configFilePath: /extra.conf
  enableMetadata: false
  variables:
    TEST: "TEST"
```

**Comment by efekarakus** [comment]
Hello @stefnpol 👋 

Which version of Copilot are you using? `copilot version`

In v1.12 we started injecting the default COPILOT* env variables: https://github.com/aws/copilot-cli/releases/tag/v1.12.0

**Comment by iamhopaul123** [comment]
Hello @stefnpol. Right now the "COPILOT*" ones are only automatically injected into the main container but not for firelens containers. So does the [`variables` field](https://aws.github.io/copilot-cli/docs/developing/environment-variables/#how-do-i-add-my-own-environment-variables). This is definitely something we should do! Thank you for catching this missing feature.

**Comment by efekarakus** [comment]
This is now released 🎉 in v1.13.0 https://github.com/aws/copilot-cli/releases/tag/v1.13.0 !

**Comment by sekamaneka** [comment]
Thanks guys. Awesome work :)
